assets/lokomotive-kubernetes/bootkube/resources/charts/lokomotive/templates/00-common.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
With no capabilities in role, we can make this service account to not mount the token by default too.